src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java

```diff
@@ -163,42 +163,31 @@ public long createNotAssigned(long userId, Levels levelName) {
                 .status(Status.STARTED)
                 .priority(Priority.LOW)
                 .build();
-
         testsRepository.save(test);
         return test.getId();
     }
 
     @Override
     public long createAssigned(long userId, Levels levelName, Instant deadline, Priority priority) {
-
         User user = usersService.getUserById(userId);
-
         restrictionsService.checkNotSelfAssign(user);
-
         restrictionsService.checkHasNoAssignedTests(user);
-
         Test test = create(user, levelName)
                 .assignedAt(Instant.now())
                 .deadline(deadline)
                 .status(Status.ASSIGNED)
                 .priority(priority)
                 .build();
-
         testsRepository.save(test);
-
         notificationService.create(NotificationType.TEST_ASSIGNED, test.getUser(), test);
-
         return test.getId();
     }
```

Comment:
Please, don't remove ALL empty lines :)
They make code more readable

Reply:
but as for me it is very difficult to read the code as on lines 173-179 =)

Reply:
Maybe let's edit IntelliJ IDEA code style properties?)
I propose these options (Editor -> Code Style -> Java -> Blank lines):

![image](https://user-images.githubusercontent.com/31935266/130367632-1845e2e5-0bff-4c93-b1fe-628a3c69c8d0.png)

Reply:
And yep, don't remove all empty lines, please))

Reply:
No, they don't. From time to time such code can be blocked by any style checkers.
Additionally, splitting a method by empty strings means that you're splitting the logic so you can split it into several methods. So, don't be shy to remove them.